src/django/api/helpers.py

```diff
@@ -82,3 +84,21 @@ def clean(column):
     if not column:
         column = None
     return column
+
+
+def value_is_in_range(range_value, range):
+    range_min = range.get('min', 0)
+    range_max = range.get('max', None)
+    value_min = range_value.get('min', 0)
+    value_max = range_value.get('max', value_min)
+    if value_min < range_min:
+        return value_max >= range_min
+    if value_min >= range_min:
+        return range_max is None or value_min <= range_max
+
+
+def convert_to_standard_ranges(number_of_workers, overlapping_ranges=set):
```

Comment:
I think using `set` instead of `set()` here initializes `overlapping_ranges` to the class `set` instead of a new `set` instance:

```py
>>> def foo(bar=set):
...     return bar
...
>>> foo()
<class 'set'>
>>> def foo(bar=set()):
...     return bar
...
>>> foo()
set()
```

However, even with that fix, I think this could potentially not work as intended because the default value of `overlapping_ranges` is initialized once and then reused each time the function is called. 

```py
>>> def add_to_set(elements, foo=set()):
...     foo.add(elements)
...     return foo
...
>>> add_to_set(10)
{10}
>>> add_to_set(20)
{10, 20}
```

More info and a workaround here: https://nikos7am.com/posts/mutable-default-arguments/